src/main/java/core/basesyntax/service/impl/ReportCreateImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportCreate;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportCreateImpl implements ReportCreate {
```

Comment:
Maybe that would be a better name
```suggestion
public class ReportWriterImpl implements ReportWriter {
```